tests/benchmark/BaseJumpRateModelV2_compiled.cairo

```diff
@@ -0,0 +1,780 @@
+%lang starknet
```

Comment:
Since it is a generated file, do we need it in the repo?

Reply:
Agreed. WIll remove it. Thanks for pointing this out.